optimize.py

```diff
@@ -195,13 +198,74 @@ def _check_missing_players(all_players, min_cost, e_raise):
                                                 str(total_report))
         raise Exception('Total missing players at price point: ' + str(miss))
 
+def create_upload_file_and_map_pids(pid_file, test_mode=False):
+    # create upload file
+    csv_name = 'test' if test_mode else 'current'
```

Comment:
I kept the test mode for writing unit tests, so no need for it if you don't plan to unit test this.